hooks/13_LoginForm/src/pages/loginPage.tsx

```diff
@@ -70,7 +70,7 @@ export const LoginPage = withStyles(styles)(withRouter<Props>(LoginPageInner));
 
 interface PropsForm {
   onLogin: () => void;
-  onUpdateField: (string, any) => void;
+  onUpdateField: (name: string, value: any) => void;
```

Comment:
Should it be `value: string` here?

Reply:
It could be for this case you are right, we are passing an string (just don't wanted to tie it up, maybe another control could use another type)